automated install/basic-install.sh

```diff
@@ -84,6 +84,17 @@ if command -v apt-get &> /dev/null; then
   LIGHTTPD_CFG="lighttpd.conf.debian"
   DNSMASQ_USER="dnsmasq"
 
+  package_check_update_available() {
+    # Check if package would be upgraded.
+    # return values:
+    #  0   if no
+    #  1   if yes
+    #  >1  if more than one package containing the name
+    #      ( might still make restarting the core package necessary )
+    ${PKG_MANAGER} -u upgrade --assume-no | grep -c "$1"
```

Comment:
`grep -c` returns a 1 via `$?` when there are 0 matches, and that trips `set -e`.